config/manager/manager.yaml

```diff
@@ -28,7 +28,8 @@ spec:
         args:
         - "--leader-elect"
         - "--provider-id-fmt=${PROVIDER_ID_FORMAT:=v2}"
-        - "--metrics-bind-addr=127.0.0.1:8080"
+        - "--diagnostics-address=${CAPA_DIAGNOSTICS_ADDRESS:=:8443}"
+        - "--insecure-diagnostics=${CAPA_INSECURE_DIAGNOSTICS:=false}"
```

Comment:
why `CAPA_DIAGNOSTICS_ADDRESS`? why not `CAPIBM_DIAGNOSTICS_ADDRESS`

 `CAPA_INSECURE_DIAGNOSTICS` => `CAPIBM_INSECURE_DIAGNOSTICS `???

Reply:
Fixed it.